pcdet/spconv_utils.py

```diff
@@ -0,0 +1,34 @@
+from typing import Set
+
+try:
+    import spconv.pytorch as spconv
+except:
+    import spconv as spconv
+
+import torch.nn as nn
+
+
+def find_all_spconv_keys(model: nn.Module, prefix="") -> Set[str]:
+    """
+    Finds all spconv keys that need to have weight's transposed
+    """
+    found_keys: Set[str] = set()
+    for name, child in model.named_children():
+        new_prefix = f"{prefix}.{name}" if prefix != "" else name
+
+        if isinstance(child, (spconv.SubMConv3d, spconv.SparseConv3d, spconv.SparseInverseConv3d)):
```

Comment:
Can we replace this by checking whether child is a instance of ``spconv.SparseConvolution``? 
Since there are so many sparse conv modules instead of these three commonly-used modules.